src/main/java/com/puppycrawl/tools/checkstyle/JavaAstVisitor.java

```diff
@@ -2127,6 +2127,11 @@ private void advanceChildToEnd() {
             }
         }
 
+        /**
+         * Returns the root node.
+         *
+         * @return the root node
+         */
         private DetailAstImpl getRoot() {
```

Comment:
How did our Javadoc check miss this? We already document private methods.

Reply:
All missing Javadoc comments are in nested private classes. This may be the cause.

Reply:
@rnveach this is expected:
https://github.com/checkstyle/checkstyle/blob/4b5cae9f7709160c9f23d0e7caf613d0b2949e65/config/checkstyle_checks.xml#L614-L617

```xml
    <module name="MissingJavadocMethod">
      <property name="allowMissingPropertyJavadoc" value="true"/>
      <property name="scope" value="private"/>
    </module>
```
After this PR the `allowMissingPropertyJavadoc` property can be set to `false`.